cypress/support/oktaLogin.js

```diff
@@ -74,6 +74,31 @@ export const login = (user) => {
 
 }
 
```

Comment:
Not sure I understand how you are loading the correct multiple mat user from environment variable? Looks like you are just going to load the same users here unless I am missing something.